contributors/devel/how-to-doc.md

```diff
@@ -56,7 +56,7 @@ Use inline links instead of urls at all times. When you add internal links to
 `http://releases.k8s.io/HEAD/<path/to/link>`. For example, avoid using:
 
 ```
-[GCE](https://github.com/kubernetes/kubernetes/blob/master/docs/getting-started-guides/gce.md)  # note that it's under docs/
+[GCE](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/getting-started-guides/gce.md)  # note that it's under docs/
```

Comment:
We may need to edit the example instead of just the link.

Reply:
ping @wackxu

Reply:
@cblecker Thanks for your kindly suggest.And I have read the file again.I agree with the opinion of @xiangpengzhao ,We should also edit the example and I have fix it.